README.md

```diff
@@ -45,17 +45,21 @@ yarn add react-semantic-ui-datepickers
 ## Usage
 
 ```jsx
-import React from 'react';
+import React, { useState } from 'react';
 import SemanticDatepicker from 'react-semantic-ui-datepickers';
 import 'react-semantic-ui-datepickers/dist/react-semantic-ui-datepickers.css';
 
-const AppWithBasic = ({ onChange }) => (
-  <SemanticDatepicker onChange={onChange} />
-);
-
-const AppWithRangeAndInPortuguese = ({ onChange }) => (
-  <SemanticDatepicker locale="pt-BR" onChange={onChange} type="range" />
-);
+const AppWithBasic = () => {
+  const [currentDate, setNewDate] = useState('');
```

Comment:
```suggestion
  const [currentDate, setNewDate] = useState(null);
```

My bad, I didn't pay attention when writing the example for you, but this initial state should be `null`.